communications/README.md

```diff
@@ -0,0 +1,9 @@
+# Communications
+
+## Async written
+
+- If you want to discuss an idea (e.g. new or existing product feature, update a marketing page), comment on or create a new story/epic in Clubhouse. @-mention specific individuals and ask them specific questions. See [Product Development](../product-development/README.md).
+- Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured. Respond to Tilters within a few days with an answer, or at least say you don't have time currently to do so.
```

Comment:
"a few days" seems like a really long time to me, especially when combined with instructions not to fall back to slack until Clubhouse has gotten in the way. I generally feel like I'm already often waiting longer than I'd like on Clubhouse replies. I don't know if I'm alone in this. It seems worth 1) polling the team on what we want to collectively commit to here, 2) adding a note that it's fine to ping people on slack for faster responses (or maybe that'd be similarly addressed by a "goals" bit at the top that says this is about establishing a source of truth rather than social stuff? I generally think a lot of process docs could greatly benefit from more discussion of the process's goals, but that's a longer conversation).

Reply:
Thanks @landism . What do you think about this?

```suggestion
- Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured. Respond to Tilters as soon as you can with an answer, or at least say you don't have time currently to do so.
```

> I generally think a lot of process docs could greatly benefit from more discussion of the process's goals, but that's a longer conversation).

@landism : This is really close to something I asked @nicks about earlier. And there's a a few comments earlier in this PR between myself and @nicks that GitHub seems to have erased (😭) which mentions this. TLDR: I've asked Nick to establish some company values written down, which should ultimately drive company process. He is thinking about it so let this comment be another reminder to Nick, i.e. Matt is indirectly interesting it this too. (I think of company values and process goals to be really close. Processes should achieve certain process goals, which should be a reflection of more fundamental values.)

In the mean time, however, I don't that to slow down improving process. So the goal of this PR is to codify existing process we are already fairly aligned with. So I think there's some value to write down things we are already doing. And if we want to start moving toward formalizing more nuanced interactions or changing process, I'll continue to push for company values.

Reply:
> I think of company values and process goals to be really close. Processes should achieve certain process goals, which should be a reflection of more fundamental values.

Hm, maybe you've got a much different experience with company values than I do. I've generally seen those as very high-level and abstract, with there still being value in discussing more proximate goals for specific processes. That seems like a bigger conversation and non-blocking for this PR, though, and maybe interesting for our next 1:1.

> So the goal of this PR is to codify existing process we are already fairly aligned with in order to set consistent expectations.

In this case, maybe we shouldn't talk about response time yet. I think we don't have a consistent expectation and response times vary greatly across the team. I personally feel like a few days is too long, but maybe most of the team doesn't, and that'd be fine. "a few days" to "as soon as possible" is a pretty significant difference when both are theoretically describing existing expectations! I actually don't know what our coworkers' existing expectations are. I was just saying that I personally think a few days is too long. I don't think my personal expectation should stand for the team's shared expectation.

I think it would be useful to come up with a time and document it here, but it's fine if you don't want to do that in this PR. I've found it very useful when we've put explicit times for exterminator response times in the working agreement. Setting it to a day or an hour really clarified how much we should worry about distractions and interrupting existing work.

As a bit of an aside, I think "as soon as possible" is not great for communicating expectations. It's rather subjective and context-sensitive. One obviously can't take it literally, so it becomes "as soon as is reasonably possible", and then some people might think a day is reasonable and other people might think two hours is unreasonable (I feel like we had a similar situation with the exterminator response time). We'll never take judgement out of the process, but objective units of time seem much more effective for aligning expectations.

Reply:
```suggestion
- Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured.
```

Reply:
Thanks @landism for raising those concerns. I just removed that sentence to resolve this PR. But very much I share your concerns and want to set these expectations properly in future PRs.

Reply:
@landism : GitHub seems to have fixed a weird bug. 🤷‍♂️ . Anyways here's the conversation I was talking about earlier: https://github.com/tilt-dev/company/pull/33#issuecomment-632211279.

> I've generally seen those as very high-level and abstract, with there still being value in discussing more proximate goals for specific processes.

I'm very interested in moving forward with something in this realm. So expect me to push for this in coming weeks and months. And we can figure out terms and definitions then.